src/main/java/data/DatabaseController.java

```diff
@@ -0,0 +1,1032 @@
+package data;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import pathfinding.Node;
+import pathfinding.ConcreteNode;
+
+import static java.sql.DriverManager.println;
+
+public class DatabaseController {
+
+	//private static String dbURL = "jdbc:derby://localhost:1527/myDB;create=true;user=me;password=mine";
+	static final String DB_URL = "jdbc:derby:FHAlpha;create=true";
+	private static String providerTable = "Provider";
+	private static String nodeTable = "Node";
+	private static String officeTable = "Office";
+	private static String neighborTable = "Neighbor";
+	private static String floorTable = "Floor";
+	// jdbc Connection
+	private static Connection connection = null;
+	private static Statement stmt = null;
+	private static ArrayList<Node> nodeList = new ArrayList<Node>();
+	private static ArrayList<Provider> providerList = new ArrayList<Provider>();
+	private static ArrayList<Floor> floorList = new ArrayList<Floor>();
+
+	//TODO: Remove main and properly initialize connections/tabes from elsewhere
+	public static void main(String[] args) {
```

Comment:
Holy mother of Cthulhu, this needs to be exterminated as soon as possible. To initialize static class members or call certain static functions (like seen here), you can use `static` instead: 

    static
    {
        createConnection();
        initializeProviderTable();
        //etc.
    }

Just make sure to take the "shutdown" call out of there too, I imagine that you probably don't want the database to die immediately after it was started up.